src/modules/staking/sora/components/ValidatorsList.vue

```diff
@@ -0,0 +1,340 @@
+<template>
+  <div class="validators">
+    <div v-if="mode !== ValidatorsListMode.RECOMMENDED" class="search-container">
```

Comment:
could be moved to getter and used later => !isValidatorModeRecommended

Reply:
Done